vulnerabilities/helpers.py

```diff
@@ -164,3 +165,34 @@ def __lt__(self, other):
         )
 
     return affected_package_with_patched_package_objects
+
+
+def split_markdown_front_matter(lines: str) -> Tuple[str, str]:
+    r"""
+    Split text into markdown front matter and the markdown body
+    Return ("", text) for text with non existing front matter
+
+    >>> text='''---
+    ... title: DUMMY-SECURITY-2019-001
+    ... description: Incorrect access control.
+    ... cves: [CVE-2042-1337]
+    ... ---
+    ... # Markdown starts here
+    ... '''
+    >>> split_markdown_front_matter(text)
+    ('title: DUMMY-SECURITY-2019-001\ndescription: Incorrect access control.\ncves: [CVE-2042-1337]', '# Markdown starts here\n')
+    """
+    fmlines = []
+    mdlines = []
+    splitter = mdlines
```

Comment:
I personally find this a little bit hard to understand and read.
What about no splitter and just `frontmatter = []` and `markdown = []` ? and then beow append directly.

Reply:
fixed